app/assets/stylesheets/account/dashboard.scss

```diff
@@ -0,0 +1,34 @@
+#navbar-mobile {
+  float: right !important;
```

Comment:
try to avoid use of `!important`
too many of those will make difficult to maintaine

Reply:
Yes, I know that it is a bad practise, but it the most 'easy way' when you work with Bootstrap template. )